frontend/src/pages/calculator/index.js

```diff
@@ -1,13 +1,247 @@
+/* eslint-disable react/jsx-props-no-spreading */
 import React from 'react';
-import { PlaceholderStyle } from '../../components/placeholderStyle/PlaceholderStyleElement';
-import './CalculatorElements';
+import axios from 'axios';
+import { useNavigate } from 'react-router-dom';
+import 'antd/dist/antd.css';
+import { Radio, Space } from 'antd';
+import {
+  FormContainer,
+  CustomSlider,
+  CustomInput,
+  BoxOne,
+  BoxTwo,
+  BoxThree,
+  BoxFour,
+  BoxFive,
+  BoxSix,
+  BoxSeven,
+  ButtonContainer,
+  CalculatorSubmitButton,
+  IllustrationContainer,
+} from './CalculatorElements';
+import Footer from '../../components/footerPurple';
+import { SectionTitle } from '../../components/sectionTitle/SectionTitle';
+import calculatorIllustration from '../../images/calculatorIllustration.svg';
 
+const formItemLayout = {
+  labelCol: { span: 24 },
+  wrapperCol: { span: 24 },
+};
 function Calculator() {
+  const [creditValue, setCreditValue] = React.useState(0);
+  const [downPaymentValue, setDownPaymentValue] = React.useState(0);
+  const [loanTermValue, setLoanTermValue] = React.useState(0);
+  const [interestRateValue, setInterestRateValue] = React.useState(0);
+  const [commissionValue, setComissionValue] = React.useState(0);
+  const [interestTypeValue, setInterestTypeValue] = React.useState(1);
+  const [installmentTypeValue, setInstallmentType] = React.useState(1);
```

Comment:
1. First of all you can `import { useState } from 'react'` to get rid of `React.` prefix.
2. I don't like this code is "copied" many times (same with handler functions below). It could be written in some smarter way. E.g. passed as a parameter when creating a new `Box`.

Reply:
solved